components/builder-web/stylesheets/base/_theme.scss

```diff
@@ -13,7 +13,7 @@ $tabs-typography: mat-typography-config(
 );
 
 $hab-theme-primary: mat-palette($md-mcgpalette0);
-$hab-theme-accent: mat-palette($md-mcgpalette2);
+$hab-theme-accent: mat-palette($md-mcgpalette1);
 $hab-theme-warn: mat-palette($md-mcgpalette5);
```

Comment:
I've changed the accent color from orange to blue to better reflect the mockups. This means I then had to override the call-to-action so that only they remained orange (Sign in with GitHub; Install GitHub App).